tutorials/epochs/plot_30_epochs_metadata.py

```diff
@@ -5,8 +5,8 @@
 ===========================
 
 This tutorial shows how to add metadata to :class:`~mne.Epochs` objects, and
-how to use :label:`Pandas query strings <indexing.query>` to select and plot
-epochs based on metadata properties.
+how to use :ref:`Pandas query strings <pandas:indexing.query>` to select and
```

Comment:
why is `pandas:` needed here but not in the same reference on line 104?

Reply:
It might not have been necessary, I just put some stuff in and once it works did not try to unify. Feel free to simplify if you want. (This might be some microseconds faster depending on how Sphinx resolves ambiguous references, but I doubt it matters. Maybe there is some advantage here in lower risk of naming collisions, too, but it also probably does not matter.)